### PR TITLE
Add section on how to add the initial admin user

### LINKED ISSUE #10531: Chapter about the initial configuration misses setup of the initial admin user
### Describe the bug

When a user wants to create a new Keycloak installation based on Quarkus, they will need to setup an initial admin user. 

This is described in the docs about containers, but not everyone uses containers. 

### Version

17.0.0

### Expected behavior

This should be part of the docs in https://www.keycloak.org/server/configuration

### Actual behavior

Section is not present.

### PR DESCRIPTION
**Pull request by ahus1**
Closes #10531

### CODE REVIEW

File: docs/guides/src/main/server/configuration.adoc [hunk 1]
```diff
@@ -154,6 +154,14 @@ is needed for deploying Keycloak in production.
 By default, the configuration options for the production mode are commented out in the `conf/keycloak.conf`. These examples
  are meant to give you an idea about the main settings to consider when running in production.
 
+== Setup of the initial admin user
+
+The initial admin user can be added manually via the web frontend when accessed from the local host or automatically via scripting using environment variables.
+
+To add the initial admin user via scripting, set the environment variables `KEYCLOAK_ADMIN` and `KEYCLOAK_ADMIN_PASSWORD` for username and password of the user.
```
Comment (DGuhr) [inline docs/guides/src/main/server/configuration.adoc:161]:
```suggestion
To add the initial admin user, set the following environment variables: `KEYCLOAK_ADMIN` for the initial admin username and `KEYCLOAK_ADMIN_PASSWORD` for the initial admin password..
```

**Comment by ahus1** [comment]
@DGuhr - thank you for the suggestions, I accepted and squashed them into my commit.

**Review by jonkoops — DISMISSED** [review]

File: docs/guides/src/main/server/configuration.adoc [hunk 2]
```diff
@@ -154,6 +154,14 @@ is needed for deploying Keycloak in production.
 By default, the configuration options for the production mode are commented out in the `conf/keycloak.conf`. These examples
  are meant to give you an idea about the main settings to consider when running in production.
 
+== Setup of the initial admin user
+
+The initial admin user can be added manually using the web frontend when accessed from localhost or automatically using environment variables.
+
+To add the initial admin user using environment variables, set `KEYCLOAK_ADMIN` for the initial admin username and `KEYCLOAK_ADMIN_PASSWORD` for the initial admin password..
```
Comment (DGuhr) [inline docs/guides/src/main/server/configuration.adoc:161]:
```suggestion
To add the initial admin user using environment variables, set `KEYCLOAK_ADMIN` for the initial admin username and `KEYCLOAK_ADMIN_PASSWORD` for the initial admin password.
```

File: docs/guides/src/main/server/configuration.adoc [hunk 3]
```diff
@@ -154,6 +154,14 @@ is needed for deploying Keycloak in production.
...
+Keycloak uses them at the first startup to create an initial user with administration rights.
+Once the first user with administrative rights exists, use the command line tool `kcadm.[sh|bat]` to create additional users.
```
Comment (DGuhr) [inline docs/guides/src/main/server/configuration.adoc:163]:
```suggestion
Once the first user with administrative rights exists, you can use the UI or the command line tool `kcadm.[sh|bat]` to create additional users.
```

**Review by DGuhr — CHANGES_REQUESTED** [review]

**Review by DGuhr — DISMISSED** [review]

**Review by DGuhr — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

**Comment by abstractj** [comment]
@ahus1 I will wait for the checks to complete, so we can get it merged.

**Pull request merged**